### PR TITLE
Feature: Support Cloud 14-day Trial

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

Feature branch to support new Cloud 14-day trial feature which include updates to Cloud Welcome email and new trial notification emails.

#### Ticket Link

```release-note
NONE
```

### CODE REVIEW

**Comment by marianunez** [comment]
@stevemudie heads up this is the feature PR for Cloud Trial for webapp, we can do a test once merged and the customer portal can create trial workspaces

**Review by nickmisasi — APPROVED** [review]

File: app/cloud.go [hunk 1]
```diff
@@ -175,3 +175,28 @@ func (a *App) SendNoCardPaymentFailedEmail() *model.AppError {
 	}
 	return nil
 }
+
+func (a *App) SendCloudTrialEndWarningEmail(trialEndDate, siteURL string) *model.AppError {
+	sysAdmins, e := a.getSysAdminsEmailRecipients()
```
Comment (ethervoid) [inline app/cloud.go:180]: For assumed convention 
```suggestion
	sysAdmins, err := a.getSysAdminsEmailRecipients()
```

**Review by ethervoid — APPROVED** [review]
LGTM! Just a minor nit

**Comment by marianunez** [comment]
@amyblais heads up this is the cloud 14 day feature that we are merging and need to cherry pick to `cloud`. The feature will be turned on/off from the portal based on the release testing this week.

**Pull request merged**